### ISSUE TITLE
groups: need ability to bulk invite users to a group

### ISSUE THREAD

**Issue by patosullivan** [tloncorp/landscape-apps#1845]
OTT

**Comment by jamesacklin** [comment]
We have a thread for this:

```
:groups &group-action [[~zod %some-group] now %fleet (~(gas in *(set ship)) ~ [~bus ~dev ~nec]) %add ~]
```

where `~zod` is the group host, `%some-group` is the group flag, `~bus`, `~dev`, and `~nec` are the ships you want to invite.

**Comment by jamesacklin** [comment]
Fixed by #1871
